Tests/SourceKitLSPTests/BackgroundIndexingTests.swift

```diff
@@ -507,11 +508,17 @@ final class BackgroundIndexingTests: XCTestCase {
         """
     )
 
-    try headerNewMarkedContents.write(
-      to: try XCTUnwrap(uri.fileURL),
-      atomically: true,
-      encoding: .utf8
-    )
+    // clangd might have Header.h open, which prevents us from updating it. Keep retrying until we get a successful
```

Comment:
As in it may be currently reading it and that has a lock :O?

Reply:
That’s what it looks like from the logs.